### DISCUSSION TITLE
苏州市养老金，29年领完才和存银行收益一样，不敢相信，有没有帮我看看算没算错

### DISCUSSION THREAD

**Discussion by LiZhiheng26** [The-Run-Philosophy-Organization/run#159] [General]
每月工资22000，交(20% 进统筹账户 8%进个人账户) 如果不交养老金存银行月利率按0.003算的

22000 * 0.28= 6160一个月 
月利息0.003 
35年是517万

个人账户 22000*0.08*12 = 21120 一年
养老金记账年记账利率 1.07  35年  292万

每月领取  ((292/139) = 2.62(个人账户养老金)) + 
（(1万月平均工资+2.2万个人工资)/2 * 35(年限) /100 = 0.29(基础养老金)) =2.38万

517万 月领取 2.38万 剩下 1.003 月利息
29年领完一开始的517本息和不交养老金存银行的收益一样

**Comment by leon-fudan** [comment]
是的，这就是养老金的定义，如果每个人都要领到比自己交的（加利息）多，这合理吗？
我看你并不符合这个repo的目标人群定义，去看一下readme吧？
